selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt

```diff
@@ -19,9 +19,3 @@ internal interface Camera {
 internal interface StillCamera : Camera {
```

Comment:
I'm thinking we should just push up the `StillCamera` interface into `Camera`. There's no need for the two layers in the hierarchy any longer.

Reply:
Right, done.